test/unit/stream_alert_rule_processor/test_parsers.py

```diff
@@ -77,18 +154,17 @@ def test_inspec(self):
 
         # load fixture file
         with open('test/unit/fixtures/inspec.json', 'r') as fixture_file:
-            data = fixture_file.readlines()
+            data = fixture_file.readline().strip()
 
-        data_record = data[0].strip()
         # setup json parser
-        parsed_result = self.parser_helper(data=data_record,
+        parsed_result = self.parser_helper(data=data,
                                            schema=schema,
                                            options=options)
 
         assert_equal(len(parsed_result), 2)
-        inspec_keys = (u'impact', u'code', u'tags', u'source_location', u'refs',
-                       u'title', u'results', u'id', u'desc')
-        assert_equal(sorted((inspec_keys)), sorted(parsed_result[0].keys()))
+        inspec_keys = ['impact', 'code', 'tags', 'source_location', 'refs',
+                       'title', 'results', 'id', 'desc']
+        assert_items_equal(parsed_result[0].keys(), inspec_keys)
```

Comment:
This is inaccurate since `assert_equal` cares about order here and we do not. Here is the result of using `assert_equal`:
```
First differing element 0:
city
name

- [u'city', u'age', u'name', u'state']
+ ['name', 'age', 'city', 'state']
```